examples/marvel/lib/src/screens/home.dart

```diff
@@ -96,7 +96,12 @@ class Home extends HookConsumerWidget {
             return Scaffold(
               appBar: AppBar(title: const Text('Error')),
               body: Center(
-                child: Text(errorMessage ?? '$err'),
+                child: Text(
+                  switch (errorMessage) {
+                    String() => errorMessage,
```

Comment:
I meant removing the `err is DioException` and that local `errorMessage` variable

Reply:
Oh, I wrongly understood. Now I have updated the code, Remi.